.github/workflows/build-workflow.yml

```diff
@@ -0,0 +1,48 @@
+## Continuous Integration Workflow
+name: Build Tests
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  pull_request:
+    branches: [release, staging]
+    paths: "src/app/**"
+
+# Target Branch Job
+jobs:
+  Build-Export:
+    # Execute on Ubuntu-Latest Distro
+    runs-on: ubuntu-latest
+    steps:
+      # Check Out Repository
+      - uses: actions/checkout@v1
+
+      # Install Node.js
+      - uses: actions/setup-node@v1.4.4
+        with:
+          node-version: 14
+          registry-url: https://npm.pkg.github.com/
+          scope: "@datature"
+
+      # Install your dependencies
+      - name: Installing
+        working-directory: ./src/app
+        run: npm install
+
+      # Build NextJS Project
+      - name: Building Nexus / NextJS
+        working-directory: ./src/app
+        run: npm run build:static
+
+      - name: Commit files
+        shell: bash
+        run: |
+          git config --local user.email "beatrice.leong98@gmail.com"
```

Comment:
@gkeechin will have to change this. what should I change this to?

Reply:
will this trigger an action recursion when the new commit triggers another cicd and so on?

Reply:
Email -
```
41898282+github-actions[bot]@users.noreply.github.com
```
User Name
```
github-actions[bot]
```